add spidertron extended support

mac3ksu opened the pull request:
Updated to sort the inventories of the SpidertronMK2 and SpidertronMK3 created by the spidertron-extended mod.

mac3ksu committed:
add spidertron extended support

Updated to sort the inventories of the SpidertronMK2 and SpidertronMK3 created by the spidertron-extended mod.

Tomeamis committed:
Fixed errors, made less redundant

Tomeamis commented:
I fixed your main branch, it works with the original spider now. I see no reason why it shouldn't work with the extended spiders, but please test it and let me know how it turns out.

mac3ksu commented:
Thank you, I will check it out.

On Wed, Mar 9, 2022 at 2:55 PM Tomeamis ***@***.***> wrote:

> I fixed your main branch, it works with the original spider now. I see no
> reason why it shouldn't work with the extended spiders, but please test it
> and let me know how it turns out.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Tomeamis/FactorioSpidertronInventory/pull/1#issuecomment-1063362179>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZFHXUXXRTI4EIRB2UK6I3U7EF3FANCNFSM5QKCXZ5A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

mac3ksu commented:
Doesn't appear to work still. Guessing it might have something to do with
how the other mod builds the entity but I'm pretty new to digging into
factorio mods. It seems like it should work. I've gone back to the v0.1.0
and changed the SpiderName to "spidertronmk2" and that doesn't work either.

On Wed, Mar 9, 2022 at 3:02 PM Marshall C ***@***.***>
wrote:

> Thank you, I will check it out.
>
> On Wed, Mar 9, 2022 at 2:55 PM Tomeamis ***@***.***> wrote:
>
>> I fixed your main branch, it works with the original spider now. I see no
>> reason why it shouldn't work with the extended spiders, but please test it
>> and let me know how it turns out.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Tomeamis/FactorioSpidertronInventory/pull/1#issuecomment-1063362179>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ABZFHXUXXRTI4EIRB2UK6I3U7EF3FANCNFSM5QKCXZ5A>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
